master/src/ComputerZoomWidget.h

```diff
@@ -38,15 +38,26 @@ class ComputerZoomWidget : public QWidget
 	ComputerZoomWidget( const ComputerControlInterface::Pointer& computerControlInterface );
 	~ComputerZoomWidget() override;
 
+	VncViewWidget* vncView() const
+	{
+		return m_vncView;
+	}
+
 protected:
+	bool eventFilter( QObject* object, QEvent* event ) override;
+
 	void resizeEvent( QResizeEvent* event ) override;
+	void closeEvent( QCloseEvent* event ) override;
 
 private:
 	void updateSize();
 	void updateComputerZoomWidgetTitle();
 
-	void closeEvent( QCloseEvent* event ) override;
+	int currentScreen;
 
 	VncViewWidget* m_vncView;
 
+Q_SIGNALS:
+	void keypressInComputerZoomWidget( );
```

Comment:
Where is this signal used?

Reply:
Ahh... Sorry. Now I see something is missing in this PR. Again it is Windows, that triggers a MouseMove event each time a key is pressed in the ComputerZoomWidget, while on Linux this does not happen. So the signal is used to reset the counter in the ComputerMonitoringWidget.

Reply:
Done.